source/cpu/cpuid_ppc64le.go

```diff
@@ -124,6 +126,8 @@ var flag2Names_ppc64le = map[uint64]string{
 	PPC_FEATURE2_DARN:           "DARN",
 	PPC_FEATURE2_SCV:            "SCV",
 	PPC_FEATURE2_HTM_NO_SUSPEND: "HTM-NO-SUSPEND",
+	PPC_FEATURE2_ARCH_3_1:       "ARCH_3_1",
+	PPC_FEATURE2_MMA:            "MMA",
```

Comment:
These look right for the P10, and I can confirm I see the expected existing labels, and the new labels. 

MMA/ARCH_3_1 show on the p10 I'm testing on. 

```# lscpu
Architecture:            ppc64le
  Byte Order:            Little Endian
CPU(s):                  16
  On-line CPU(s) list:   0-15
Model name:              POWER10 (architected), altivec supported```

Reply:
based on this test result, for ppc64le platform
/lgtm